pkg/generate/testdata/models/apis/mq/0000-00-00/generator.yaml

```diff
@@ -0,0 +1,9 @@
+ignore:
+  resources:
+    - Configuration
+    - User
+resources:
+  Broker:
+    hooks:
+      sdk_update_pre_build_request:
```

Comment:
Could you please add a test for sdk_delete_pre_request as well so I can pick up this change as soon as it's merged?

Reply:
Yes, I will. I am currently modifying this patch to support `template_path` for hook configurations.

Reply:
@surajkota please note I've updated this PR to include support for specifying `template_path` in addition to raw `code`